lumin/app.package

```diff
@@ -1,4 +1,6 @@
 DATAS = "digest.sha512.signed" : "." \
         "bin/" : "bin/" \
-        "resources/" : "resources/"
+        "../resources/" : "resources/" \
```

Comment:
I think we no longer need to put `resources` folder in here (since we moved the `resources` inside `assets`, right?